nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/test/resources/test-tag-service-override.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.tag.service.override">
+
+  <require>org.nuxeo.tag.service.properties.override</require>
+  <require>org.nuxeo.ecm.platform.tag.TagService</require>
```

Comment:
I don't think you need these requires. The test should deploy things in the proper order anyway.